src/exposeHidTransport.js

```diff
@@ -0,0 +1,18 @@
+import TransportNodeHid from "@ledgerhq/hw-transport-node-hid-noevents";
+
+let transport = null;
+
+export default app => {
+  app.exposeFunction("ledgerHidTransport", async (cmd, ...args) => {
+    if (cmd === "open") {
+      transport = await TransportNodeHid.create();
```

Comment:
```suggestion
      transport = await TransportNodeHid.open("");
```

is less intense.

i also suggest to add an `invariant(!transport, )` before the open() , to avoid cases where you call close() and open() and the close will set transport back to null, making the transport never working again (because as soon as device is opened, it's locked)